tests/testdata/additional-seeds-two-rack-four-node-dc.yaml

```diff
@@ -8,7 +8,8 @@ spec:
   serverVersion: "6.8.1"
```

Comment:
did you mean to `git mv` the old file, or to `cp && git add`?

Reply:
oops.  Will fix.

Reply:
Ok, that was a good catch.  I just added a copy from master.  Once this is merged to master, the older file history should win because of how git tracks content.

Reply:
squash and merge ftw :)